x/staging/posts/client/cli/query.go

```diff
@@ -75,12 +72,12 @@ func GetCmdQueryPost() *cobra.Command {
 func GetCmdQueryPosts() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "posts",
```

Comment:
I think it's much better to require the user to specify a subspace. This will block them from being able to performing high-intensive queries like fetching all the posts. 

```suggestion
		Use:   "posts [subspace]",
		Args: os.ExactArgs(1),
```